pkg/services/publicdashboards/api/middleware.go

```diff
@@ -50,7 +50,7 @@ func RequiresExistingAccessToken(publicDashboardService publicdashboards.Service
 		}
 
 		// Check that the access token references an enabled public dashboard
-		exists, err := publicDashboardService.AccessTokenExists(c.Req.Context(), accessToken)
+		exists, err := publicDashboardService.EnabledPublicDashboardExistsByAccessToken(c.Req.Context(), accessToken)
```

Comment:
Yes! Thank you for tackling this.

What do you think about `PublicDashboardEnabledExistsByAccessToken` ? 

@owensmallwood thoughts?

Reply:
I can change it but in that case I will change `EnabledPublicDashboardExistsByDashboardUid ` too.

Reply:
Works for me!